distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java

```diff
@@ -120,6 +121,35 @@ private void startManagedService(Configuration sourceConfig, Map<String, Object>
 
         bundleCtx.registerService(ConfigurationListener.class.getName(), listener, null);
 
+        long waitForService = 0;
```

Comment:
I would suggest extracting this new chunk of code into a method to wait for a service.